yijing/iching_export.md

```diff
@@ -95,7 +95,7 @@
 {\Large 元亨，利牝馬之貞。君子有攸往，先迷後得主，利。西南得朋，東北喪朋。安貞吉。}
 \end{center}
 
-> Primal offering, bounty in the mare&#x27;s divinity. The superior person has a destination, initially lost then obtaining guidance, advantage. Southwest to obtain friendship; northeast to lose friends. Peaceful divination auspicious.
```

Comment:
@azhrcs - I actually hand translated this one

> 利牝馬之貞

I'm actually viewing "steadfastness" as a red flag that it's a translation that needs review - it's Han/Jesuit/German moralizing for `貞`? I actually like "the mare's divinity" .. open to other interpretations .. in this case "steadfastness" actually fits because it's "the mare" and I think that is related to the divinity of the mare 🤔